website/views/shared/profiles/stats/active-pet-and-mount.jade

```diff
@@ -0,0 +1,5 @@
+div(ng-show='user._id != profile._id')
+  h4
+  table.table.table-striped
+    +basicRow('activePet', '{{::profile.items.currentPet ? (isBasicAnimal(profile.items.currentPet, \'pet\') ? env.t(\'petName\', {potion: Content.hatchingPotions[petPotion].text(), egg: Content.eggs[petEgg].text()}) : env.t(Content.specialPets[profile.items.currentPet])) : env.t(\'noActivePet\')}}')
```

Comment:
@khipkin The way to get the localized string for a pet/mount is:

 Type | Pets | Mounts
------ | ----- | --------
Basic | env.t(\'petName\', {potion: Content.hatchingPotions[petPotion].text(), egg: Content.eggs[petEgg].text()}) | isBasicAnimal(profile.items.currentMount, \'mount\') ? env.t(\'mountName\', {potion: Content.hatchingPotions[mountPotion].text(), mount: Content.eggs[mountEgg].text()}
Special | env.t(Content.specialPets[profile.items.currentPet]) | env.t(Content.specialMounts[profile.items.currentMount])

I was having trouble making an external method that does that (it wouldn't return anything), so I modified what #5422 had.